aidial_adapter_bedrock/llm/model/stability/v2.py

```diff
@@ -0,0 +1,205 @@
+        last_message = self._validate_last_message(messages)
+        # Handle text content
+        match last_message.content:
+            case str(text):
+                text_prompt = text
+            case list():
+                text_parts = []
+
+                for part in last_message.content:
+                    match part:
+                        case MessageContentTextPart(text=text):
+                            text_parts.append(text)
+                        case MessageContentImagePart(
+                            image_url=ImageURL(url=url)
+                        ):
+                            image_resources.append(
+                                URLResource(
+                                    url=url,
+                                    supported_types=SUPPORTED_IMAGE_TYPES,
+                                )
+                            )
+                        case _:
+                            assert_never(part)
+                if text_parts:
+                    text_prompt = " ".join(text_parts)
+            case None:
+                pass
+            case _:
+                assert_never(last_message.content)
+
+        if (
+            last_message.custom_content
+            and last_message.custom_content.attachments
+        ):
+            image_resources.extend(
+                [
+                    AttachmentResource(
+                        attachment=attachment,
+                        supported_types=SUPPORTED_IMAGE_TYPES,
+                    )
+                    for attachment in last_message.custom_content.attachments
+                ]
+            )
+
+        if not self.image_to_image_supported and image_resources:
+            raise ValidationError(
+                f"Image-to-image is not supported for {self.model}"
+            )
+        if len(image_resources) > 1:
+            raise ValidationError("Only one input image is supported")
+        try:
+            response, _ = await self.client.ainvoke_non_streaming(
+                self.model,
+                remove_nones(
+                    {
+                        "prompt": text_prompt,
+                        "image": (
+                            (
+                                await image_resources[0].download(self.storage)
+                            ).data_base64
+                            if image_resources
+                            else None
+                        ),
+                        "mode": (
+                            "image-to-image"
+                            if image_resources
+                            else "text-to-image"
+                        ),
+                        "output_format": "png",
+                        "strength": 0.5 if image_resources else None,
+                    }
+                ),
+            )
+        except self.client.client.exceptions.ValidationException as e:
+            raise ValidationError(e.response["Error"]["Message"]) from e
```

Comment:
Let's move the exception handling logic to `aidial_bedrock_adapter.server.exceptions`

Reply:
I've actually decided to drop this exception handling completely, and rely on our default `to_dial_exception` decorator

The reason I introduced this, is because without this part error message looked like this
```
An error occurred (ValidationException) when calling the InvokeModel operation: {"detail":"width must be between 640 and 1536"}
```

When with this code this reduced to
```
{"detail":"width must be between 640 and 1536"}
```

And I just don't think that this `An error occurred (ValidationException) when calling the InvokeModel` is completely useless